pkg/client/applyconfiguration/job.min.io/v1alpha1/commandspec.go

```diff
@@ -1,5 +1,5 @@
 // This file is part of MinIO Operator
-// Copyright (c) 2023 MinIO, Inc.
+// Copyright (c) 2024 MinIO, Inc.
```

Comment:
Shouldn't this be 2023-2024?

Reply:
Should it?, it's an easy change but not sure, so far we keep a single year in header

Reply:
Okay, if we do it like this everywhere then it's fine. I'm just used to have that starting year in there as well. But that may be hard for generated files anyway...